Sources/BeeTracking/TrackingLikelihoodModel.swift

```diff
@@ -57,6 +57,25 @@ public struct TrackingLikelihoodModel<Encoder: AppearanceModelEncoder, FG:Genera
               backgroundModel : BG(from: bgFeatures, given:p?.backgroundModel))
   }
   
+  /**
+   Train only the probabilistic models from a collection of foreground and background patches
+   - Parameters:
+    - encoder: the encoder object
+    - foregroundPatches: [...,H,W,C] batch of foreground patches to train with
+    - backgroundPatches: [...,H,W,C] batch of background patches to train with
+    - p: optional hyperparameters.
+   */
+  public init(with encoder: Encoder,
```

Comment:
see my comment above. I think a training a likelihood model with a given encoder is a different beast. Not a required change in this PR but we should discuss.